### PR TITLE
Fix/sidekiq schedule

### PR DESCRIPTION
**Pull request by moustachu**

### CODE REVIEW

**Review by armandfardeau — APPROVED** [review]

File: config/sidekiq.yml
Comment (Quentinchampenois) [inline config/sidekiq.yml:13]: Here comment are not representative of the current cron config 
'0 0 0 * * *' => "At 12:00 AM, on day 1 of the month"
https://crontab.cronhub.io/

File: config/sidekiq.yml
Comment (Quentinchampenois) [inline config/sidekiq.yml:17]: Here comment are not representative of the current cron config 
'0 0 1 * * *' => "At 12:00 AM, on day 1 of the month
https://crontab.cronhub.io/

File: config/sidekiq.yml
Comment (Quentinchampenois) [inline config/sidekiq.yml:21]: Here comment are not representative of the current cron config 
'0 0 18 * * *' => "At 12:00 AM, on day 18 of the month"
https://crontab.cronhub.io/

**Review by Quentinchampenois — COMMENTED** [review]
Hi, 

Are we sure about the cron configuration, according to the defined format, the cron configuration is not representative of the comment with it 

cron config : 


minute (0-59) | hour (0 - 23) | day of the month (1 - 31) | month (1 - 12) | day of the week (0 - 6)
-- | -- | -- | -- | --

**Comment by moustachu** [comment]
@Quentinchampenois Well that's the qwirk on Cron expressions
Depending on the software you can find expressions with 5 to 7 arguments.
The most used is the 5 arguments version, like the one you have in the `crontab` on your linux system.

From what I gather, here is the breakdown : 

image

([source](https://ppolyzos.com/2020/05/09/understanding-cron-expressions/))

**5 arguments**
minute (0-59) | hour (0 - 23) | day of the month (1 - 31) | month (1 - 12) | day of the week (0 - 6)
-- | -- | -- | -- | --

**6 arguments**
second (0-59) | minute (0-59) | hour (0 - 23) | day of the month (1 - 31) | month (1 - 12) | day of the week (0 - 6)
-- | -- | -- | -- | -- | --

**7 arguments**
second (0-59) | minute (0-59) | hour (0 - 23) | day of the month (1 - 31) | month (1 - 12) | day of the week (0 - 6) | year (1970 - 2099)
-- | -- | -- | -- | -- | -- | --

The one used in the sidekiiq schedule gem is a 6 arguments version.
And we did test that worked that way

File: config/sidekiq.yml
Comment (moustachu) [reply config/sidekiq.yml:17]: Your generator doesn't give me the same answer ...
image

File: config/sidekiq.yml
Comment (moustachu) [reply config/sidekiq.yml:21]: Your generator doesn't give me the same answer ...
image

**Comment by moustachu** [comment]
BTW I made a mistake on this PR : it was meant to be merge with `release/0.24-stable`

So I will put it on hold to be able to merge it in 0.24 before resolving the conflict and merge this one

**Review by Quentinchampenois — APPROVED** [review]

**Pull request merged**
